test/unit/lib/format-time.test.js

```diff
@@ -0,0 +1,167 @@
+const format = require('../../../src/lib/format-time');
```

Comment:
Yay tests! But they'll need updating for the new requirements.

Reply:
Updated -- there are far fewer now because the functionality is simpler.